.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6-dev"
```

Comment:
Seems to work fine for me https://github.com/pypa/pip/blob/master/.travis.yml#L20-L21